docs/tutorial/django.rst

```diff
@@ -383,12 +383,12 @@ We can now stop the container and remove the corresponding image:
     :end-before: [docs:stop-docker-updated-end]
     :dedent: 2
 
-Reset your environment
+Reset the environment
 ======================
```

Comment:
When you change headings you need to change the number of characters for the underline to match the length of the heading (otherwise the CI will fail anyway)

Reply:
Thanks for catching!

Reply:
I also made the same mistake in the FastAPI tutorial, so I updated that one as well.